docs/next.config.js

```diff
@@ -141,6 +141,7 @@ module.exports = withTM({
     return map;
   },
   reactStrictMode,
+  distDir: 'dist',
```

Comment:
Based on the nature of the problem that we are trying to solve: scoping. I think that we should bring more context. Maybe something like this: 
```suggestion
  distDir: '_next/toolpad',
```

Reply:
I think the `basePath` option is meant for exactly this purpose, scoping.

Reply:
https://github.com/vercel/next.js/issues/31718#issuecomment-1185690997